src/Lecture1.hs

```diff
@@ -108,7 +113,8 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+strSum :: string -> Int
```

Comment:
Looks like the code fails to compile because there's a type: a small `s` is used instead of a big one `S`
```suggestion
strSum :: String -> Int
```